java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -1365,6 +1365,8 @@ JNIEXPORT jint JNICALL Java_ai_rapids_cudf_ColumnVector_getNativeNumChildren(JNI
         return static_cast<jint>(column->num_children() - 1);
       } else if (column->type().id() == cudf::type_id::STRUCT) {
         return static_cast<jint>(column->num_children());
+      } else {
+        return 0;
       }
```

Comment:
Realized I am on an older version of gcc which may be why I didnt see this, will update! Dont want compile errors to pass thru like this.